tap_appstore/streams.py

```diff
@@ -1,72 +1,150 @@
 """Stream type classes for tap-appstore."""
 
 from __future__ import annotations
+import tempfile
 
-import sys
-import typing as t
+import os
+from applaud.connection import Connection
+from applaud.endpoints.sales_reports import SalesReportsEndpoint
+from singer_sdk import Stream, typing as th
+import datetime
+from datetime import datetime
+import logging
 
-from singer_sdk import typing as th  # JSON Schema typing helpers
+logger = logging.getLogger(__name__)
 
-from tap_appstore.client import AppStoreStream
 
-if sys.version_info >= (3, 9):
-    import importlib.resources as importlib_resources
-else:
-    import importlib_resources
+class SalesReportStream(Stream):
+    name = "sales_reports"
+    #primary_keys = ["id"]
+    schema = th.PropertiesList(
```

Comment:
Let's keep camel case  so we don't have any change from the previous schema https://github.com/Automattic/tap-appstore/blob/master/tap_appstore/schemas/sales_report.json

Reply:
Done in [66c1df6](https://github.com/Automattic/tap-appstore-singer-sdk/pull/1/commits/66c1df62aec466567bf67159957965f7307623f0)